### ISSUE TITLE
img alt

### ISSUE THREAD

**Issue by ghost** [ACodingGuy/teacozy#4]
https://github.com/ACodingGuy/teacozy/blob/dec4471b9657f8f16115a83b0683e3bbf078c778/index.html#L37-L53

Nice job giving all  of your `img` tags a descriptive `alt` attribute, just in case your website is not able to load it! 👍 

Doing so will help improve the accessibility of your website when it's visited by visually impaired users on screen readers-- as well as any other time that your website is just unable to render images.
